RevenueCatUI/CustomerCenter/Data/PurchaseInformation.swift

```diff
@@ -236,7 +240,7 @@ struct PurchaseInformation {
                 self.isExpired = !isActive
 
             case .nonSubscription:
-                self.isLifetime = true
+                self.isLifetime = subscribedProduct?.productType == .nonConsumable
```

Comment:
fyi in android I changed this to `isSubscription`, because I was getting confused with lifetime promotionals